src/main/java/com/smartbear/readyapi/client/execution/SmartestApiWrapper.java

```diff
@@ -17,5 +18,7 @@
 
```

Comment:
sure, this was created when we were still trying few names for test server. I will rename.